docs/_temp/Installation.md

```diff
@@ -0,0 +1,146 @@
+
+Based on such, we want to measure their productivity and stability. This is how [DORA](docs/DORA.md) does that:
+- Productivity:
+  - How many times does the team `deploy`? (a.k.a. [Deployment Frequency](docs/Metrics/DeploymentFrequency.md))
+  - How fast are the `pull requests` resolved? (a.k.a. [Lead Time](docs/Metrics/LeadTimeForChanges.md))
+- Stability:
+  - How many `incidents` per `deploys` does the team have? (a.k.a. [Change Failure Rate](docs/Metrics/CFR.md))
+  - How fast are these `incidents` solved? (a.k.a. [Median Time to Restore](docs/Metrics/MTTR.md))
+
+All these questions/metrics are based on either `pull requests`, `deployments`, or `incidents`.
+Note: all three are completely separate entities and associated only to their project.
+
+#### General advice
+
+There are 3 red lines when it comes to structuring your DevLake `projects`:
+
+But when we scale this up, a few problems arise:
+- A team usually works with multiple `repositories`
+- A team also might work on different projects, and we want to measure these projects separately (e.g. it is not the same to work on a big old legacy than on a greenfield)
+- There may be multiple teams
+- A `board` contains incidents of multiple teams or projects
+- A `repository` is managed by multiple teams or projects, e.g. a monorepo
+- A `pipeline` can trigger deployments in multiple repositories
+- Some organizations want to measure DORA based on projects, and some want to measure it by teams
+
+This is where the `project` concept comes to play.
+
+## 2. What is a DevLake project?
+In the real world, a project is something being built and/or researched to solve some problem or to open new grounds.
+A DevLake project is a grouping of `pull requests`, `deployments`, or `incidents`. It can be seen as a real-world project or product line. DevLake measures DORA metrics for each project.
+
+![](project_pipeline.png)
+
+_Note: It does not matter if a team works on a particular repository more than another.
+The metrics are calculated over the entire set of repos, and the values are accumulated.
+More on that: [Debugging DORA Issue Metrics](docs/Troubleshooting/Dashboard.md#debugging-dora-issue-metrics)_
+
+## 3. As a team lead, how many DevLake projects do I need?
+
+Because of its simplicity, the concept is flexible: you decide how to arrange `pull requests`, `deployments`, and `incidents`
+either by your specific projects, by teams, technology, or any other way.
+
+The examples below show the patterns of how to organize your projects.
+To keep things simple we assume that we work with **GitHub** repos, **Jira** boards, 
+and **Jenkins CI/CD** deploys in each GitHub repo.
+
+The same would apply to other repos (e.g. GitLab or BitBucket), boards (e.g. TAPD), 
+or CI/CD (e.g. GitLab CI, Azure DevOps).
+
+## 4. Building use case 1
+
+There are `2 teams` with `2 boards`, 3 `repos`, and 3 `cicd pipelines`. 
```

Comment:
Hi @KucherenkoSerhiy , I'm not sure how you define `team` or `project`.

From my perspective, a `team` is an organizational concept (people-wise); while a `project` is a project (work-wise). Therefore, in this use case, we might use `2 projects` instead of `2 teams`. 

Also, to make it more real, you can use the example of real-life open-source projects, for example, `DevLake` and `[DevStream](https://github.com/devstream-io/devstream)`. DevLake is a project that manages 3 repos, `apache/incubator-devlake`, `apache/incubator-devlake-website`, and `apache/incubator-devlake-helm-chart`.

I hope it makes sense.

Reply:
Hey @Startrekzky, thank you for the tips! Sure, 2 projects sound more coherent to DevLake. 
I also left a side note that for DORA there are no `teams`, only `projects`.

Regarding DevLake, I agree and also think it should come first as a base use case. The one already in the document is rather an expansion, introducing intersections between projects.